catalog/transformer/catalog-transformer-streaming-lib/src/test/java/org/codice/ddf/transformer/xml/streaming/lib/TestSaxEventToXmlElementConverter.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.transformer.xml.streaming.lib;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+
+import javax.xml.stream.XMLStreamException;
+
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+public class TestSaxEventToXmlElementConverter {
+
+    /*
+     * Some of the "reconstructedExpectations" in this class are different than the input. This is acceptable and expected,
+     * because the output just has to be semantically the same as the input, it doesn't have to be literally the same.
+     * For example, sometimes the namespaces are declared at different scopes, in a different order, comments are removed, etc.
+     */
+
+    @Test
+    public void testSaxEventToXmlElementConverterRedeclaredNamespaceUri()
+            throws XMLStreamException, IOException, SAXException {
+        String doubleDeclaredNamespaceUriSnippet =
+                "<x xmlns:n1=\"foobar\" \n" + "   xmlns=\"foobar\">\n"
```

Comment:
Could you either reformat these to make them easier to read or just move all of them to test resources so they can be correctly formatted xml?